src/engine/strat_engine/filesystem.rs

```diff
@@ -5,15 +5,22 @@
 use std::path::{Path, PathBuf};
 use std::process::Command;
 
-use devicemapper::DM;
-use devicemapper::{ThinDev, ThinDevId, ThinStatus, ThinPoolDev};
+use devicemapper::{Bytes, DM, DataBlocks, ThinDev, ThinDevId, ThinStatus, ThinPoolDev};
+use devicemapper::consts::SECTOR_SIZE;
+
+use nix::sys::statvfs::statvfs;
+use nix::sys::statvfs::vfs::Statvfs;
 
 use super::super::engine::{Filesystem, HasName, HasUuid};
 use super::super::errors::{EngineError, EngineResult, ErrorEnum};
 use super::super::types::FilesystemUuid;
 
+use super::thinpool::DATA_BLOCK_SIZE;
+
 use super::serde_structs::{FilesystemSave, Recordable};
 
+pub const FILESYSTEM_LOWWATER: DataBlocks = DataBlocks(256);
```

Comment:
Up until now we have been using one "W" as, ```..._LOWATER```. I assumed that was a devicemapper spelling convention. If it is, we should stick with it. If not, we should fix the others.